cmd/examples/user-encrypt/main.go

```diff
@@ -1,41 +1,53 @@
 package main
 
 import (
-	"flag"
+	"encoding/json"
+	"io/ioutil"
 	"log"
-	"os"
+	"path/filepath"
+	"runtime"
 
-	"github.com/jinfwhuang/ds-toolkit/go-pkg/cmd-utils"
-	"github.com/urfave/cli/v2"
-)
-
-var AppFlags = []cli.Flag{
-	cmd_utils.GrpcPort,
-	cmd_utils.LogLevel,
-}
-
-var (
-	privateKey = flag.String("private_key", os.Getenv("PRIVATE_KEY"), "A secp256k1 private key in hex form, prefixed with 0x")
+	ethereum "github.com/ethereum/go-ethereum/crypto"
+	"github.com/jinfwhuang/ds-toolkit/go-pkg/ds"
 )
 
 const (
-	// secp256k1 keys
-	testPrivkeyHex = "0x12de257b783b96ce90012a6c45f3ce61216dd60f22159d2f5cb9e17f3126bbe5"
+	// secp256k1 key
+	testPrivkeyHex = "12de257b783b96ce90012a6c45f3ce61216dd60f22159d2f5cb9e17f3126bbe5"
 )
 
-func init() {
-	log.SetFlags(log.Llongfile)
+func fileInRuntimeDir(file string) string {
+	_, filename, _, _ := runtime.Caller(0)
+	return filepath.Dir(filename) + file
 }
 
 func main() {
-	//ctx := context.Background()
-	flag.Parse()
-	if *privateKey == "" {
-		*privateKey = testPrivkeyHex
+	alicePrivateKey, err := ethereum.HexToECDSA(testPrivkeyHex)
+	if err != nil {
+		panic(err)
 	}
 
-	log.Println("fff")
-}
-
+	exampleData, err := ioutil.ReadFile(fileInRuntimeDir("/data.json"))
+	if err != nil {
+		log.Fatal("could not find data file")
+	}
 
+	dataBlob, err := ds.CreateDataBlob(exampleData, &alicePrivateKey.PublicKey)
+	if err != nil {
+		panic(err)
+	}
+	dataBlobJson, err := json.Marshal(dataBlob)
```

Comment:
Looks good.

I would recommend print out a few more comments about what is happening. For example:
1. The input data is
2. The encrypted datablob is in this file
3. We are able to recover the data

Just some simple print statements so that when someone run the program, ` go run ./cmd/examples/user-encrypt/`, they can see the standard out and get a sense of what is happening.